pyindi/www/static/js/updater-indi.js

```diff
@@ -294,6 +294,29 @@
 		var vector = updater.vector(indi, appendTo);
 
 		return vector;
+	},
+	/**
+	 * Handles the delProperty tag. 
+	 * @param {String} device name of indi device.
+	 * @param {String} name name of indi property.
+	 *
+	 */
+	delete(device, name) {
+		if(Config.CUSTOM_GUI) {
+		var deviceSelector = `[data-custom-device="${device}"]`;
+		var vectorSelector = `[data-custom-vector="${name}"]`;
+		}
+		else {
+
+			var deviceSelector = `[data-device="${device}"]`;
+			var vectorSelector = `[data-vector="${name}"]`;
+		}
+		var selector = `${deviceSelector}${vectorSelector}`; 
```

Comment:
If we build the `indi` payload as suggested in the previous comment, we need to make these changes. 

```suggestion
        delete(indi) {
        var selector;
        if (builder.customGui) {
          var deviceSelector = `[data-custom-device="${device}"]`;
 		  var vectorSelector = `[data-custom-vector="${name}"]`;
 		  selector = `${deviceSelector}${vectorSelector}`;
 		}
 		else {
 		  selector = `#${generateId.vector(indi)}`;
```

My reason for this is to use the `generateId` functions to select by ID, as there _could_ be multiple elements with the same `data-device` and `data-name` values. An ID would make sure we get the one and only. 

`builder.customGui` is a boolean value that is automatically flipped on or off depending on the `initialize` function in `indi.js`. I would like to move away from the `Config.CUSTOM_GUI` as I don't think it is the best way to proceed.